lib/junipersrx.py

```diff
@@ -28,6 +28,19 @@
 import logging
 
 
+def junipersrx_list(name, data):
+  return '%s [ %s ];' % (name, ' '.join(data))
+
+
+class IndentList(list):
+  def __init__(self, indent, *args, **kwargs):
```

Comment:
I moved the IndentList class below the error definitions and added a newline before the def